src/modules/aggsig/tests_impl.h

```diff
@@ -190,41 +210,52 @@ void test_aggsig_api(void) {
             CHECK(secp256k1_ec_pubkey_create(ctx, &pub_nonces[j], sec_nonces[j]) == 1);
         }
 
+        /* Randomize keys */
+        for (j = 0; j < 2; j++) {
+            random_scalar_order_test(&tmp_s);
+            secp256k1_scalar_get_b32(seckeys[i], &tmp_s);
+            CHECK(secp256k1_ec_pubkey_create(ctx, &pubkeys[i], seckeys[i]) == 1);
+        }
+
```

Comment:
Thanks for spotting it, now the test passes without problems. Since we are already running 20 iterations of this test, I think it makes sense to generate new secret keys for each one. Otherwise we're testing mostly the same thing 20 times.